S21/ajayrazz99/refactorme.py

```diff
@@ -1,84 +1,74 @@
 import unicodedata
 
-class Word:
 
-  def __init__(self, word, bcpCode, std=False):
-    self._w = word
-    self._l = bcpCode
-    self._finalSigma = False
-    self._standardIrishSpelling = std
-    # OLD EXPERIMENTAL CODE for dealing with vowel harmony
-    # self._numVowels = 0
-    # for c in word:
-    #   if c in 'aeiouAEIOU':
-    #   self._numVowels += 1
+class CheckBCP_47:
```

Comment:
I'd rather see a class that represents a BCP-47 string, with a constructor and methods, independent of the Word class.